src/helpers/types.ts

```diff
@@ -69,6 +69,7 @@ export interface TrancheTypeEnum extends Enum {
 export interface NavDetails extends Struct {
   value: u128
   lastUpdated: u64
+  // values: Vec<ITuple<[u64, u128]>>
```

Comment:
@filo87 do you know why this throws a type error when using the plural `values` here? It's not actually used in the code base but I saw that it was in the lookup file

Reply:
I think the issue here is that `.values` is a reserved default object method from javascript. So it cannot be assigned. The name property name is reserved and cannot be used. 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Object/values

Reply:
thanks! Would it make sense to remove the line since it's not used?

Reply:
Yup makes sense ;)